src/service/action-impl.js

```diff
@@ -200,6 +203,10 @@ export class ActionService {
         this.addChangeDetails_(event);
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'input-debounced') {
+      this.root_.addEventListener('input', debounce_(event => {
+        this.trigger(dev().assertElement(event.target), name, event);
```

Comment:
Thanks for the heads up. Since the `if` blocks above have the same caveat, and this is only used to trigger amp actions I think it should be safe for this use-case.